src/components/MenuLink/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { MenuLink as ReachMenuLink } from '@reach/menu-button';
+
+const noop = () => {};
+
+const MenuLink = ({ onSelect, children, isDisabled, href, as }) => (
+  <ReachMenuLink
+    className={isDisabled ? 'fdsMenuItem--disabled' : undefined}
+    onSelect={!isDisabled ? onSelect : noop}
+    aria-disabled={isDisabled}
+    href={href}
+    as={as}
+  >
+    {children}
+  </ReachMenuLink>
+);
+
+MenuLink.propTypes = {
+  /** selection callback */
+  onSelect: PropTypes.func.isRequired,
+
+  /** React children - any JSX can be used in a menu item */
+  children: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.node), PropTypes.node])
+    .isRequired,
+
+  /** Styles menu item as disabled and disables selection events */
+  isDisabled: PropTypes.bool,
+
+  href: PropTypes.string,
+
+  /** Useful for passing react router Link */
+  as: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
```

Comment:
soo..... `MenuLink` accepts rest props. `href` isn't explicitly defined and is merely passed as a rest prop, HOWEVER. I feel like it's important to specify it explicitly here else people are going to be confused. However, the `as` prop IS explicitly defined, and it'll be obvious to whoever uses that, that a `to` prop can obviously be passed without having to spell it out, and without incurring upfront confusion ("question: what's the difference between href and to"). Not married to this though and happy to make changes.

Reply:
What do you think of my suggestion above (documenting both `href` and `to` explicitly)?

Reply:
I'm cool with that, I'll make that change. I may need to circle back to see if I did what you were expecting though.